src/components/document-metadata-modal.yml

```diff
@@ -7,9 +7,6 @@ cloneDocument:
 editDocument:
   en: Edit document metadata
   de: Dokument-Metadaten bearbeiten
-roomRequired:
-  en: Please specify a room
-  de: Bitte gib einen Raum an
```

Comment:
Did you delete this validation text on purpose (because the label of the field is more or less the same)?

Reply:
Initially I even thought that the label was wrong because it was phrased like a validation message 😅 So I reduced them.

Reply:
So, as we agreed, we'll bring back this validation message and rephrase the label to

* eng: "Room"
* ger: "Raum"

Reply:
Aye aye, done!

Reply:
Cool, thanks!